### PR TITLE
Update FAudio to v19.09

### PR DESCRIPTION
**Pull request by LeandroStanger**
The new Proton 4.11-4 shipped with FAudio 19.09.

### CODE REVIEW

**Comment by nanonyme** [comment]
Needs rebase against beta

**Comment by LeandroStanger** [comment]
bot, build com.valvesoftware.Steam

File: resources/freedesktop-sdk.ld.so.blockedlist [hunk 1]
```diff
@@ -90,5 +90,5 @@ Proton\ 3.16\ Beta/dist/bin/wine64 Proton\ 3.16\ Beta/dist/lib64/libFAudio.so
 Proton\ 3.16\ Beta/dist/bin/wine64-preloader Proton\ 3.16\ Beta/dist/lib64/libFAudio.so
 Proton\ 4.2/dist/bin/wine64 Proton\ 4.2/dist/lib64/libFAudio.so.0.19.06
 Proton\ 4.2/dist/bin/wine64-preloader Proton\ 4.2/dist/lib64/libFAudio.so.0.19.06
-Proton\ 4.11/dist/bin/wine64 Proton\ 4.11/dist/lib64/libFAudio.so.0.19.08
-Proton\ 4.11/dist/bin/wine64-preloader Proton\ 4.11/dist/lib64/libFAudio.so.0.19.08
+Proton\ 4.11/dist/bin/wine64 Proton\ 4.11/dist/lib64/libFAudio.so.0.19.09
```
Comment (nanonyme) [inline resources/freedesktop-sdk.ld.so.blockedlist]: Do you mean by this that Proton version 4.11 stopped shipping libFAudio.so.0.19.09?

Comment (gasinvein) [reply resources/freedesktop-sdk.ld.so.blockedlist]: We probably shouldn't remove previous versions, they are still there for users who didn't update Proton yet.

Comment (nanonyme) [reply resources/freedesktop-sdk.ld.so.blockedlist]: This is now obsoleted since commit was removed as unnecessary after recently merged PR.

**Pull request merged**